1_create_db.md

```diff
@@ -0,0 +1,74 @@
+```
+Câu 1: Tạo database như hình trên (sau khi tạo thì import database, sẽ gửi sau)
+CREATE DATABASE `train_mysql` DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
+CREATE TABLE IF NOT EXISTS `user`(
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
```

Comment:
khoá chính thì nên dùng không âm nhé em, em apply cho tất cả các bảng còn lại nhé
`id` INT UNSIGNED NOT NULL AUTO_INCREMENT,